.gitignore

```diff
@@ -2,3 +2,4 @@
 /.nyc_output/
 /coverage/
 /node_modules/
+*Key.json
```

Comment:
Consider our env file for this, if the value is not too ugly and the process not too much of a hassle.

Reply:
let's keep it as is for now.